docker/templates/django/{{cookiecutter.app_name}}/requirements.txt

```diff
@@ -0,0 +1,14 @@
+# Pin these requirements to the latest versions when you create a new project,
+# e.g. Django==3.0.2.
```

Comment:
I don't love that this step is required for template users, but I couldn't think of another way of doing this that didn't require us to be constantly updating this file when the packages we use get major updates.

Reply:
Awesome idea, I added some instructions for using pip-chill to pin dependencies.